main.go

```diff
@@ -141,6 +153,11 @@ func main() {
 		fmt.Println(grayscale(14)(k+":"), cyan(strings.Join(v, ",")))
 	}
 
+	// If -head flag set, then skip other output
+	if onlyHeader {
```

Comment:
What about the timings ? Would it make sense to  do this check earlier and skip reading the body (it's ok just to let the connection close when the program exits).

Suggestion:

add this check at line 140, "if !onlyHeader { ... read body }`

and switch the `resp.Body.Close` to a defer